src/steps/lido/lido-wrap-steth-step.ts

```diff
@@ -0,0 +1,66 @@
+import { RecipeERC20AmountRecipient, StepConfig, StepInput, StepOutputERC20Amount, UnvalidatedStepOutput, RecipeERC20Info } from "../../models";
+import { Step } from "../../steps/step";
+import { compareERC20Info } from "../../utils";
+import { LidoWSTETHContract } from "../../contract/lido";
+import { Provider } from "ethers";
+
+export class LidoWrapSTETHStep extends Step {
+    readonly config: StepConfig = {
+        name: "Lido Wrap stETH",
+        description: "Wrap stETH to wstETH",
+        hasNonDeterministicOutput: false
+    };
+
+    readonly wstETHTokenInfo: RecipeERC20Info;
+    readonly stETHTokenInfo: RecipeERC20Info;
+    private provider: Provider;
+
+    constructor(wstETHTokenInfo: RecipeERC20Info, stETHTokenInfo: RecipeERC20Info, provider: Provider) {
+        super();
+        this.wstETHTokenInfo = wstETHTokenInfo;
+        this.stETHTokenInfo = stETHTokenInfo;
+        this.provider = provider;
+    }
+
+    private async getWrapAmount(stakeAmount: bigint): Promise<bigint> {
+        const wstETHContract = new LidoWSTETHContract(this.wstETHTokenInfo.tokenAddress, this.provider);
+        const wrappedAmount = await wstETHContract.getWstETHByStETH(stakeAmount);
+        return wrappedAmount;
+    }
```

Comment:
I think you can do this, can you check?
```suggestion
    private async getWrapAmount(stakeAmount: bigint): Promise<bigint> {
        const wstETHContract = new LidoWSTETHContract(this.wstETHTokenInfo.tokenAddress, this.provider);
        return await wstETHContract.getWstETHByStETH(stakeAmount);
    }
```